ppx/test/pp_ocaml.expected

```diff
@@ -41,7 +41,9 @@ let make =
                  (Props : < .. >  Js_of_ocaml.Js.t) (fun x -> x#name)) in
     fun ?name ->
       fun ?key ->
-        fun () -> React.create_element make (make_props ?key ?name ()))
+        fun () ->
+          React.set_display_name make __FUNCTION__;
```

Comment:
I wonder if there is a way to call `set_display_name` from outside this function, to avoid wasted work. Or is it necessary to do so every time a new element is created?

Reply:
It most likely could, but I'm struggling to figure out where that would be in the ppx code. We could probably also mitigate the impact by only emitting this in dev mode, but I'm not sure how to do that either. Yet.

Reply:
Fixed in https://github.com/ml-in-barcelona/jsoo-react/pull/166/commits/d0cc8c38d163be2ec2ae86279c03d95ad93753eb